pkgs/tools/package-management/nix/default.nix

```diff
@@ -50,7 +50,10 @@ in lib.makeExtensible (self: {
 
   nix_2_5 = throw "nixVersions.nix_2_5 has been removed";
 
-  nix_2_6 = throw "nixVersions.nix_2_6 has been removed";
+  nix_2_6 = common {
```

Comment:
Maybe we should add a comment pointing for the bug so we don't remove this until we have it fixed.

Reply:
Yes, please otherwise the next person it going to drop it again.

Reply:
I added a comment